scap/scc/map.jinja

```diff
@@ -20,7 +20,7 @@
         'cmd' : 'C:\\Program Files\\SCAP Compliance Checker ' ~ version ~ '\\cscc.exe',
         'content_cache' : opts['cachedir'] ~ '\\files\\base\\' ~ scap.content.source | replace('salt://', '') | replace('/', '\\'),
         'retry_script' : 'salt://' ~ tpldir ~ '/files/retry.bat',
-        'version' : version,
+        'version' : '~ version ~',
```

Comment:
This doesn't seem right to me... this would create the literal string, `~ version ~`...

Reply:
I think on line 9, can maybe instead just pipe to the `string` filter... e.g. 

```jinja
{%- set version = salt.pillar.get('scap:lookup:scc:version', '') | string -%}
```

Reply:
Ok.  Let me check.  For some reason, I was still seeing an error without ''.

Reply:
> I think on line 9, can maybe instead just pipe to the `string` filter... e.g.
> 
> ```
> {%- set version = salt.pillar.get('scap:lookup:scc:version', '') | string -%}
> ```

I tested this suggestion locally and looks to be working.  I'll push an update and kick off another build to fully verify.